pallets/attestation/src/lib.rs

```diff
@@ -127,7 +127,7 @@ pub mod pallet {
 		type Currency: Currency<AccountIdOf<Self>> + ReservableCurrency<AccountIdOf<Self>>;
 
 		/// The deposit that is required for storing an attestation.
-		#[pallet::constant]
+		// FIXME: Add back #[pallet::constant] once the fix is merged
```

Comment:
https://github.com/KILTprotocol/substrate/commit/f15c85e09b56a9d7a317e856ae324c47ea924ece

Since there is a deposit in Delegations and Attestations directly depends on delegations, it's not clear which deposit should be used inside the macro

Reply:
https://github.com/paritytech/substrate/pull/9942